yang/iproute2-ip-link.yang

```diff
@@ -31,6 +31,42 @@ module iproute2-ip-link {
     * Typedefs
     */
 
+    typedef ipv4-with-cider{
+    // we can't use inet, as libyang does subnetting to inet:ipv4-prefix.
+        type string {
```

Comment:
How does it work with FRR ? I do not understand how to connect the dots, but what's about:
  https://github.com/FRRouting/frr/blob/master/yang/libyang_plugins/frr_user_types.c